numcodecs/blosc.pyx

```diff
@@ -86,24 +87,31 @@ except ImportError:
 _importer_pid = os.getpid()
 
 
-def init():
+def _init():
     """Initialize the Blosc library environment."""
     blosc_init()
 
+init = deprecated(_init)
```

Comment:
I guess decorators don't work in .pyx files?

Reply:
They probably do, but the idea here was to leave a non-deprecated version of the functions (starting with `_`) that we can call internally, and a deprecated version which has the same name as previous.